api/client/stats_helpers.go

```diff
@@ -24,6 +24,7 @@ type containerStats struct {
 	NetworkTx        float64
 	BlockRead        float64
 	BlockWrite       float64
+	PidsCurrent      uint64
```

Comment:
That's setting cgroup value to the api type on daemon side. `PidsCurrent` is client side value used for printing.